Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -86,6 +85,47 @@ function calculateUserState() {
     }
 }
 
+enum ClientModule {
+    Immunization = "Immunization",
+    Medication = "Medication",
+    Laboratory = "Laboratory",
+    Encounter = "Encounter",
+    Comment = "Comment",
+    CovidLabResults = "CovidLabResults",
+    Dependent = "Dependent",
+    Note = "Note",
+}
+
+function getAvailableModules() {
+    const availableModules: string[] = [];
+    const configModules = store.getters["config/webClient"].modules;
+    if (configModules.Comment) {
```

Comment:
this can be more succint:
```
const availableModules: string[] = [];
    const configModules: { [id: string]: boolean } =
        store.getters["config/webClient"].modules;

    for (const moduleName in configModules) {
        if (configModules[moduleName]) {
            availableModules.push(moduleName);
        }
    }

Reply:
A lambda using filter would work as well

Reply:
nice one!